Antonio_Ramos_Documentation

```diff
@@ -140,12 +148,86 @@ new code - pFC = r"G:\MAX-Filer\Collab\Labs-kbuzard-S18\Admin\CARR\California\Ba
 and got shapefile exception again when trying to run the script 
 
+06/08/2020
 - realized I was working on the wrong script and started working on the K-function Local_3Stage.py located inG:\MAX-Filer\Collab\Labs-kbuzard-S18\Admin\kfunctions
 
 - once I started working on the correct script I realized that the Will's dosumentation is not up to date and does not reference the same files as the script does. So, I created an excell spreadsheet to list all the files mentioned by Will in his documentation in an attemtp to better understand where everything was coming from, what it did, and how it should function with the python scripts. 
 
+
 - emailed Will asking if he had any more updated documentation. Will responded with the most comprehensive documentation that he has, however it still does not mention the files being refrenced by the K-function Local_3Stage.py script. 
 
+06/09/2020
 - I will now begin doing a line by line analysis of the K-function Local_3Stage.py script to better understand what it is doing with the data files being referenced 
 
 - All modules necessary for running K-function Local_3Stage.py installed onto computer to  C:\Apps-SU\Anaconda3\pkgs. Able to fully run trial version of 3 stage script, K-function Local_3Stage Trial.py, however when proceeding to try and run the K-function Local_3Stage.py script all modules are imported and programs runs effectively until it stars performing an endless loop of simulations at line 125. Printing the following over and over again "SIMULATION #### Threw and error." with "####" being respective simulation numbers.
+
+06/10/2020
+- going to open up data files referenced in K-function Local_3Stage.py in ArcMap to see if the data they contain properly corresponds to the data structures being described in the K-function Local_3Stage.py script 
+
+- cannot open ArcMap from windows start bar, continue getting notification that the application is not responding. Will attempt to open from the command prompt window  
+
+- finally able to open arcmap by starting it from the start bar and waiting for the program to repsond 
+
+- when adding the data that is referenced in the K-function Local_3Stage.py program to a blank map in ArcMap the block data (CA_Block_New.shp) appears well, however, the zip data (CA_ZCTA_New.shp) does not appear on the map and it seems that when I add it as a data layer nothing changes on the map, furthermore, the labs data (GoodLabs_New.shp) does not appears as points on the block data map but as one point far away from the block data map. 
+
+- evaluated attribute tables to ensure that the proper data column names in the table were being referenced by the K-function Local_3Stage.py program. All data table names are correctly referenced, however, there is also another attribute "geometry" which is being listed on lines 39-41 and I am not sure what that is doing, it does not appear in any of the attribute tables from the data files. 
```

Comment:
This is a pretty good description of what a "geometry" is: https://geopandas.org/data_structures.html

Reply:
Thank you, very helpful